.github/workflows/conan.yml

```diff
@@ -84,7 +84,11 @@ jobs:
           Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
           refreshenv
           conan profile detect
-          conan create .
+          conan create . `
+            --build=missing `
+            -o "&:shared=False" `
+            -s:h compiler.cppstd=17 `
+            -s:h compiler.runtime=static
```

Comment:
@acmorrow I _think_ this (or some subset of it) is necessary for or else I get
```ERROR: There are invalid packages:
grpc/1.69.0: Invalid: grpc/1.69.0 shared not supported by Visual Studio
```
in CI.